src/App.tsx

```diff
@@ -1,20 +1,29 @@
-import { PeoplePage } from './components/PeoplePage';
-import { Navbar } from './components/Navbar';
-
 import './App.scss';
+import { Navigate, Route, Routes } from 'react-router-dom';
+import { PeoplePage } from './pages/PeoplePage';
+import { HomePage } from './pages/HomePage';
+import { NotFoundPage } from './pages/NotFoundPage';
+import { NavBar } from './components/NavBar';
+
+export const App = () => (
+  <div data-cy="app">
+    <NavBar />
+
+    <main className="section">
+      <div className="container">
+        <Routes>
+          <Route path="/" element={<HomePage />} />
+
+          <Route path="home" element={<Navigate to="/" replace />} />
```

Comment:
It's good practice to have some kind of variable that stores all route values in one place, enum, or just a const. It'll be very clear to see all existing routes in one place and also when in some other components you have to use a redirect or link you can import that variable and avoid using hardcoded values which usually cause bugs and makes refactoring harder.